includes/degree-functions.php

```diff
@@ -619,6 +620,63 @@ function get_degree_admission_requirements_modern_layout( $degree ) {
 }
 
 
+/**
+  * Returns HTML for the news and spotlight section.
+  *
+  * @since 3.4.0
+  * @author RJ Bruneel
+  * @param object $degree WP_Post object representing a degree
+  * @return string HTML markup
+  */
+function get_degree_news_spotlight_modern_layout( $degree ) {
+	$degree_news_shortcode = get_field( 'degree_news_shortcode', $degree );
+	$degree_spotlight = get_field( 'degree_spotlight', $degree );
+
+	if( empty( $degree_news_shortcode ) && empty( $degree_spotlight ) ) return '';
+
+	ob_start();
+	?>
+
+	<div class="container py-lg-5">
+		<div class="row justify-content-between align-items-end">
+			<div class="col-auto">
+				<h2 class="h1 mb-0">In The News</h2>
+			</div>
+			<div class=" col-auto">
+				<p class="mb-0">
+					<a class="h6 text-uppercase mb-0 text-secondary" href="https://www.ucf.edu/news/">
```

Comment:
Can we add the `text-default-aw` class to this link? We do this in more recent adaptions of this style 'In The News' section, and I think it helps to visually separate the h2 and the link